useful-scripts/docs/copy-trello.md

```diff
@@ -0,0 +1,21 @@
+## Copy Trello board script
+
+This script copies a Trello board and replaces the references on the new board's cards' checklists with references to cards
+
+Coded in Swift 3.2 using [Marathon](https://github.com/JohnSundell/Marathon)
+
```

Comment:
Add a comment clarifying it may not work in ubuntu, and who to ask in case of need.